jetty-util/src/main/java/org/eclipse/jetty/util/URIUtil.java

```diff
@@ -470,8 +470,7 @@ public static String decodePath(String path, int offset, int length)
                             char u = path.charAt(i + 1);
                             if (u == 'u')
                             {
-                                // TODO remove %u support in jetty-10
-                                // this is wrong. This is a codepoint not a char
+                                // UTF16 encoding is only supported with UriCompliance.Violation.UTF16_ENCODINGS
```

Comment:
What about the comment about removing %u support on line 560 in this file too?

https://github.com/eclipse/jetty.project/blob/a09d9e94b5cc542b8436f9ef3171da42cfc3b13b/jetty-util/src/main/java/org/eclipse/jetty/util/URIUtil.java#L560-L562

Reply:
This comment doesn't seem to apply here, as `UriCompliance.Violation.UTF16_ENCODINGS` are not even used in this URIUtil class?